packages/cli/src/create/helpers/create-app.ts

```diff
@@ -141,6 +160,23 @@ export async function createApp({
       );
     }
 
+    const jsonFile = path.join(root, "package.json");
+    const packageString = fs.readFileSync(jsonFile);
+    try {
+      const packageJson = JSON.parse(packageString.toString());
+      if (packageJson) {
+        packageJson.name = appName;
+      }
+
+      fs.writeFileSync(jsonFile, JSON.stringify(packageJson, null, 2) + os.EOL);
+
+      console.log("Updated package.json.");
+      console.log();
+    } catch (e) {
+      console.log("Reading JSON error");
```

Comment:
We should handle the error here.

I don't think failing to rename the package name is severe enough to exit the process, but we should maybe just console.log this - "Something went wrong renaming package, skipping..."